datastore/cloud-client/snippets.py

```diff
@@ -16,8 +16,128 @@
 import datetime
 from pprint import pprint
 
-from google.cloud import datastore
 import google.cloud.exceptions
+from google.cloud import datastore
```

Comment:
You could add "# noqa: I100" to the end of this line (and perhaps the line before it) to suppress the lint message and declare that the order is intentional, not an error.